pandas/_libs/tslibs/tzconversion.pyx

```diff
@@ -332,7 +332,10 @@ timedelta-like}
     # Shift the delta_idx by if the UTC offset of
     # the target tz is greater than 0 and we're moving forward
     # or vice versa
-    first_delta = info.deltas[0]
+    if len(info.deltas):
```

Comment:
The problematic cases here are with zoneinfo tzs right?  Those should be going through the branch on L292, so looks like `is_zoneinfo` is going wrong

Reply:
woops, got that condition flipped. never mind